Zend/tests/bug79897/bug79897.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+bug79897: Promoted constructor params with attribs cause crash
+--FILE--
+<?php
+
+@@Attribute
+class B {
+    public function __construct($value)
+    {
+    }
+}
+
+class A {
+    public function __construct(
+        @@B(12) public $b
+    )
+    {
+    }
+}
+
+echo "file compiled successfully";
```

Comment:
It would be more useful to check the actual attribute arguments here:
```
var_dump((new ReflectionParameter(['A', '__construct'], 'b'))->getAttributes()[0]->getArguments());
var_dump((new ReflectionProperty('A', 'b'))->getAttributes()[0]->getArguments());
```
Also, please add a closing `?>` here and drop the extra directory from the test (just Zend/tests/bug79897.phpt is enough).

Reply:
Alright, I've made the requested changes to this test. It definitely feels better to have the assertion actually asserting something specific 👍